javascript-this/javascript-this-quiz-notes.md

```diff
@@ -0,0 +1,66 @@
+# javascript-this-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What is `this` in JavaScript?
+it's an implicit parameter
```

Comment:
Take another shot at this after Q&A

Reply:
I wrote more but I moved it to the following question
i'll fix it